### PR TITLE
Fix cryptic error when creating new cubes.

### LINKED ISSUE #33: Cannot edit plans
Accessing properties  of plan or structures results in

```
Traceback (most recent call last):
  File "/home/bassler/.local/lib/python2.7/site-packages/pytripgui/leftmenu.py", line 210, in plan_properties_field
    pub.sendMessage("gui.field.open", field)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/arg1/publisher.py", line 36, in sendMessage
    topicObj.publish(data)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/arg1/publishermixin.py", line 23, in publish
    self._publish(data)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/topicobj.py", line 383, in _publish
    self.__sendMessage(data, topicObj, iterState)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/topicobj.py", line 397, in __sendMessage
    self._mix_callListener(listener, data, iterState)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/arg1/publishermixin.py", line 34, in _mix_callListener
    listener(self, data)
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/lib/pubsub/core/arg1/listenerimpl.py", line 51, in __call__
    cb(msg, **kwargs)
  File "/home/bassler/.local/lib/python2.7/site-packages/pytripgui/main.py", line 116, in load_dialog
    pytripDialog.Init(msg.data)
  File "/home/bassler/.local/lib/python2.7/site-packages/pytripgui/dialogs/fielddialog.py", line 41, in Init
    self.check_isocenter.Bind(wx.EVT_CHECKBOX, self.on_check_isocenter_changed)
AttributeError: 'NoneType' object has no attribute 'Bind'
```

### LINKED ISSUE #59: Creating cube on loaded DICOM fails with cryptic error
Linux, pytripgui 1.2.2 + pytrip 1.3.1

Loaded some sample DICOM from https://subversion.assembla.com/svn/slicerrt/trunk/SlicerRt/data/ and tried to create a on it (File -> Create cube).

Following error appeared:

screenshot_2017-02-13_18-23-51

### PR DESCRIPTION
**Pull request by nbassler**
Fix #59, broken .xrc file and data
Fix #33, plans could not be edited.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+pytripgui/_version.py export-subst
```
Comment (grzanka) [inline .gitattributes]: Could you please revert this one ?

Comment (grzanka) [inline .gitattributes]: @nbassler please revert again

Comment (nbassler) [inline .gitattributes]: i just did, it keeps apprearing again and again! :(

Comment (grzanka) [inline .gitattributes]: :/ use IDE ;)

Comment (nbassler) [inline .gitattributes]: jaja schon gut.

**Pull request merged**